crates/rustc_codegen_spirv/build.rs

```diff
@@ -11,6 +11,9 @@ use std::process::{Command, ExitCode};
 use std::{env, fs, mem};
 
 /// Current `rust-toolchain.toml` file
+/// WARNING!!! cargo-gpu is new relying on this being a string literal! It will
```

Comment:
Quick typo here, I think it should be "cargo-gpu is _now_"...

Reply:
fixed